source/index.html

```diff
@@ -1,12 +1,208 @@
+        </svg>
+      </a>
+      <nav class="main-nav">
+        <!-- Кнопка на мобильной версии -->
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Открыть меню.</span>
+        </button>
+        <button class="main-nav__toggle" type="button">
+          <span class="visually-hidden">Закрыть меню.</span>
+        </button>
+        <!-- Само меню -->
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">Каталог товаров</a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="#">Вязание на заказ</a>
+          </li>
+        </ul>
+        <form class="search" action="https://echo.htmlacademy.ru/" method="get">
+          <svg class="search__icon" width="18" height="18" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" aria-label="Искать товар в магазине «Мишка»." role="img">
```

Comment:
```suggestion
          <svg class="search__icon" width="18" height="18" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false" role="img">
```

чёто видимо в первый раз не заметила. Не надо описание, тк скринридер сюда не попадёт. Он дальше прочтет лейбл, так что всё ок. Вот это прям явный пример, что картинка не нужна не зрячим, и поэтому от них скрывается как не нужная и не несущая никакой инфы. Лейбл прекрасно справляется.

А потом на деске, ты будешь скрывать лейбл, как вижуалли-хидден, тогда он будет продолжать читаться скрин-ридерами, но не будет виден.